City-of-Helsinki/drupal-asuntotuotanto
# Drupal core update 10.3.8

### Description

Drupal core and composer security updates


### How to test

- make fresh
- login as admin
- go search api indexing
  - rebuild track information and index both indexes
- go check hitas /& haso listing pages
- open some project and check that everything works (template side no need login and sent application)
